dnf/module/repo_module_version.py

```diff
@@ -61,20 +61,21 @@ def _install_profiles(self, profiles, defaults_used):
             if profile not in self.profiles:
                 self.report_profile_error(profile, defaults_used)
 
-            for single_nevra in self.profile_nevra(profile):
-                subject = Subject(single_nevra)
-                nevra_obj = list(subject.get_nevra_possibilities(hawkey.FORM_NEVR))[0]
-
-                self.base.install(single_nevra, reponame=self.repo.id, forms=hawkey.FORM_NEVR)
-                self.base._goal.group_members.add(nevra_obj.name)
+            for nevra_object in self.profile_nevra_objects(profile):
+                nevr = "{}-{}".format(nevra_object.name, nevra_object.evr())
+                nevra = "{}.{}".format(nevr, nevra_object.arch)
```

Comment:
installed package is picked by depsolver and doesn't have to match with nevra
not a blocker at the moment, but we need to improve this in the future